pkgs/development/compilers/crystal/default.nix

```diff
@@ -56,6 +56,8 @@ let
       inherit sha256;
     };
 
+    outputs = [ "out" "lib" "bin" "share" ];
```

Comment:
"share" is a little bit unusual. For instance, we don't have a lib.getShare, like we do for lib.getDev, lib.getLib, and lib.getBin. Unless the share & out outputs are intentionally separate, I would recommend just putting share in $out.

Reply:
"doc" would be a little bit better if you would like.

Reply:
`share` has stuff like shell completions, the manpage, a bunch of html docs, and the license... I'm not sure it matters since this gets all symlinked into `$out` anyway, but if you feel strongly about it i don't mind removing it either.